use/identifying_to_services.rst

```diff
@@ -181,19 +181,26 @@ Services plugin
 ---------------
 
 The Services plugin comes with Supybot and should be an easy way to
-identify your bot, but SASL and ``username:password`` as server password
-are recommended over it. Start by loading Services with::
+identify your bot, but SASL is recommended over it. Start by loading
+Services with::
```

Comment:
`username:network` is not supported on OFTC and may confuse users by not being universal and as Services doesn't require global password nowadays, I think it's safe to drop that part?